src/CoreBundle/private/modules/CMSInterface/CMSInterface.class.php

```diff
@@ -25,10 +28,16 @@ public function Execute()
         }
 
         parent::Execute();
-        $aGroupListUser = $oGlobal->oUser->oAccessManager->user->groups->GetGroupListAsArray();
-        $aGroupListUser = TTools::MysqlRealEscapeArray($aGroupListUser);
+        /** @var SecurityHelperAccess $securityHelper */
+        $securityHelper = ServiceLocator::get(SecurityHelperAccess::class);
+
+        $groups = $securityHelper->getUser()?->getGroups();
+        $aGroupListUser = TTools::MysqlRealEscapeArray($groups);
         $sGroupSQL = '';
         foreach ($aGroupListUser as $sGroupId => $sGroupSystemName) {
+            if ('-' === $sGroupId) {
```

Comment:
Why was this added?

Reply:
because I was thinking roles, not groups (every user has the ROLE_CMS_USER with no real id). removed